examples/example_plugin/example_plugin/filter_extensions.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+
+from nautobot.extras.plugins import PluginFilterExtension
+from nautobot.utilities.filters import MultiValueCharFilter
+
+
+def suffix_search(queryset, name, value):
+    return queryset.filter(description=f"{value[0]}.nautobot.com")
+
+
+class TenantFilterExtension(PluginFilterExtension):
```

Comment:
Seems to me it would be good to have some brief yet descriptive docstrings for these?